contactme.css

```diff
@@ -0,0 +1,160 @@
+body {
```

Comment:
Consider putting shared styles across all pages into one central css file that's imported across all pages, and then unique styles into a page-specific css file